include/libhal/servo/rc.hpp

```diff
@@ -1,9 +1,14 @@
+// Use #pragma once is as an include guard for headers
```

Comment:
```suggestion
// Use "#pragma once" as an include guard for headers
// This is required because ...
```

Explain in more detail why it is we choose to use pragma once.